libs/sql-schema-describer/tests/describers/postgres_describer_tests.rs

```diff
@@ -1,7 +1,6 @@
 mod cockroach_describer_tests;
 
 use crate::test_api::*;
-use barrel::{types, Migration};
```

Comment:
the changes in this file are just some opportunistic cleanups, not related to the fix